README.md

```diff
@@ -9,13 +9,168 @@ DUH component export to Scala
 npm i duh-scala
 ```
 
+To export a scala blackbox wrapper and attach method. This is a scala `LazyModule`
+that has diplomatic nodes corresponding to the bus interfaces of the duh component.
 ```bash
 duh-export-scala <mycomp>.json5 -o <output-dir>
 ```
 
+To export a `RegisterRouter` corresponding the the `memoryMap`s in a duh component.
+A `RegisterRouter` is an abstract scala description of memory-mapped registers that
+is generic to any bus interface. Concrete implementations for TileLink and AXI4 are
+included in the generated scala.
+```bash
+duh-export-regmap <mycomp>.json5 -o <output-dir>
+```
+
+To export a scala blackbox wrapper and attach method for a monitor. This is a
+special kind of chisel `Module` that can be attached to diplomatic edges
+corresponding to the monitor's bus interface type.
+```bash
+duh-export-monitor <mycomp>.json5 -o <output-dir>
+```
+
 ## API
+The hierarchy of the output modules is looks like
+```
+top: N${name}Top
+└── imp: L${name}
+    └── blackbox: ${name}
```

Comment:
Is it `L${name}Top` or `L${name}`? Are those two separate classes or are they the same class? If they are the same class, can we make sure the text-based tree diagram includes both? In the paragraphs below, it's a little bit confusing to see the both `L${name}Top` and `L${name}` used but only one of them described up here, since as a reader I'm not sure if they are two separate concepts or the same concept.